src/main/java/org/folio/des/scheduling/quartz/job/bursar/OldDeleteJob.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.des.scheduling.quartz.job.bursar;
+
+import static org.folio.des.scheduling.quartz.QuartzConstants.TENANT_ID_PARAM;
+
+import java.util.Date;
+
+import org.folio.des.config.FolioExecutionContextHelper;
+import org.folio.des.service.JobService;
+import org.folio.spring.scope.FolioExecutionContextSetter;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@RequiredArgsConstructor
+@Log4j2
+public class OldDeleteJob implements org.quartz.Job {
+
+  private final JobService jobService;
+  private final FolioExecutionContextHelper contextHelper;
+  private static final String PARAM_NOT_FOUND_MESSAGE = "'%s' param is missing in the jobExecutionContext";
+
+  @Override
+  public void execute(JobExecutionContext jobExecutionContext) throws JobExecutionException {
+    String tenantId = getTenantId(jobExecutionContext);
+    var current = new Date();
+    try (var context = new FolioExecutionContextSetter(contextHelper.getFolioExecutionContext(tenantId))) {
+      jobService.deleteOldJobs();
+    }
+    log.info("execute:: deleteOldJobs executed at: {}", current);
```

Comment:
Timestamp is present in logs anyway, don't think there's value in adding current date